auslib/db.py

```diff
@@ -526,9 +526,26 @@ def forUpdate(self, rowData, changed_by):
         row['timestamp'] = self.getTimestamp()
         return self._insertStatement(**row)
 
-    def getChange(self, change_id, transaction=None):
-        """ Returns the unique change that matches the give change_id """
-        changes = self.select(where=[self.change_id == change_id], transaction=transaction)
+    def getChange(self, change_id, key=None, value=None, data_version=None, transaction=None):
```

Comment:
change_id should be an optional parameter now; callers shouldn't be forced to pass it when they're quering by data_version.